package-lock.json

```diff
@@ -1,215 +1,392 @@
 {
   "name": "msgpack-rpc-node",
   "version": "0.1.0-alpha.3",
```

Comment:
Could you please update the package-lock.json since you've updated the version in package.json.